### ISSUE TITLE
Autoconfigure default MessageVerifierReceiver and MessageVerifierSender implementations for Kafka

### ISSUE THREAD

**Issue by OlgaMaciaszek** [spring-cloud/spring-cloud-contract#1897]

**Comment by AdrianHarenczyk** [comment]
The former implementation was deleted in commit: fa51d6a076be8c5dc620ea6e226ad6c87dc2c0ac Maybe by a mistake? @marcingrzejszczak

**Comment by marcingrzejszczak** [comment]
No, we removed all the mock versions of receivers. This issue is about doing autoconfigs like the ones created [here](https://github.com/spring-cloud-samples/spring-cloud-contract-samples/blob/main/producer_kafka_middleware/src/test/java/com/example/BaseClass.java#L92-L132) or [here](https://github.com/spring-cloud-samples/spring-cloud-contract-samples/blob/main/consumer_kafka_middleware/src/test/java/com/example/ApplicationTests.java#L90-L104).

**Comment by AdrianHarenczyk** [comment]
So the issue is currently, that Kafka has no implementation to verify contracts. The developer adding 'org.springframework.cloud:spring-cloud-starter-contract-verifier' needs to write his own implementation of Receivers, as they are gone, and there is no notice about it. They are gone from version 4 at least, so people migrating to spring boot 3.2 need to also write their own implementation of kafka **KafkaStubMessages** because without it all their contract tests related to kafka fail. If that is just what was intended, please point me to the place where those crucial parts of the code are implemented, as otherwise I need to just write that part on my own. 
![image](https://github.com/spring-cloud/spring-cloud-contract/assets/38125398/1393155e-75ed-4e92-8180-c60d7d6d0bc1)
Here is the stack issue about it: https://stackoverflow.com/questions/77661101/spring-boot-3-migration-spring-cloud-starter-contract-verifier-and-messageverifi

**Comment by marcingrzejszczak** [comment]
That's correct, we write about this in the [migration path](https://github.com/spring-cloud/spring-cloud-contract/wiki/Spring-Cloud-Contract-4.0-Migration-Guide#removing-support-for-mocked-amqp-oob-amqp-and-oob-kafka)

**Comment by AdrianHarenczyk** [comment]
Thanks a lot, that is pretty much clear now.

**Comment by fatihdogmus** [comment]
hello, is there any plans to implement this feature? it would be nice if spring itself can provide a real kafka based verifier.
